front/components/LP/instruction.tsx

```diff
@@ -9,10 +9,10 @@ export default class Instruction extends React.Component<InstructionProps, any>
     return (
       <div className="instruction">
         <div className="heading">
-          Schedule senior design presentation&nbsp;in&nbsp;5&nbsp;minutes.
+          How to schedule your senior design presentation&nbsp;in&nbsp;under&nbsp;5&nbsp;minutes
```

Comment:
I want to have short and concise sentences for here. Current sentence fits in two lines with any device. So I would like to keep that.